spec/lib/account_deleter_spec.rb

```diff
@@ -107,11 +108,18 @@
     end
   end
 
+  describe "#delete_user_invitation_code" do
+    it 'deletes user invitation code' do
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.